CHANGELOG.md

```diff
@@ -5,12 +5,32 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 since 0.4.1, and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased](https://github.com/freedomofpress/dangerzone/compare/v0.7.1...HEAD)
+## [0.8.0](https://github.com/freedomofpress/dangerzone/compare/v0.8.0...0.7.1)
```

Comment:
At first I thought it was weird to have an empty "Unreleased" section, but I understand how this can be useful. We could also have the section added post-release, but because I don't think we don't have this step right now, let's keep it as you propose :+1:

Reply:
Done in 3e5e3c0aa7d4a9e56699517e6e16894112636ddd